packages/url/src/test/index.test.js

```diff
@@ -24,6 +24,13 @@ describe( 'addQueryArgs', () => {
 
 		expect( addQueryArgs( url, args ) ).toBe( 'https://andalouses.example/beach?night=false&sun=true&sand=false' );
 	} );
+
+	test( 'should update args to an URL with array parameters', () => {
```

Comment:
Do we use `test` or do we use `it` ?

Reply:
😅I was not even aware it exists

Reply:
We use both. I'm more interested in which one we should settle on. I don't care which it is, just that it's consistent.

Reply:
I think we use `it` more, so probably better to settle on this one?

Reply:
`test` is default if you check Jest docs 🤷‍♂️